be/src/common/config.h

```diff
@@ -1428,4 +1428,11 @@ CONF_mInt32(thrift_max_frame_size, "16384000");
 // allows for structures nested up to 64 levels deep.
 CONF_mInt32(thrift_max_recursion_depth, "64");
 
+// if turned on, each compaction will use at most N segments,
+// such as `max_cumulative_compaction_num_singleton_deltas` segments
+// for now, only support non-pk LAKE compaction.
+CONF_mBool(enable_compaction_strict_segment_count_check, "false");
```

Comment:
enable_partial_segments_compaction?

Reply:
changed to enable_lake_compaction_use_partial_segments